workflow-testing/src/test/java/com/squareup/workflow1/WorkerStressTest.kt

```diff
@@ -66,6 +67,7 @@ class WorkerStressTest {
     }
   }
 
+  @ObsoleteCoroutinesApi
```

Comment:
Nit: Tests are never obsolete or experimental themselves, they should always opt in.